src/main.rs

```diff
@@ -1796,6 +1796,7 @@ fn get_setup() -> Setup {
             normalisation_release_cf,
             normalisation_knee_db,
             ditherer,
+            position_update_interval: None,
```

Comment:
Hmm, if this is always `None` then there is also no need to add Handling in `player_event_handler.rs`. Now the question, does it make sense to enable this by default in the binary? Maybe with a one second duration?

Reply:
Yet couldn't determine the usecase for the events with the standalone binary. 1sec sounds reasonable

Reply:
Yeah make sense, I guess then we would have three possible solution

1. The only use-case I would currently see is when the `onevent` functionality is used. So maybe it makes more sense to only enable the position update when `onevent` is in use?
2. We could also just remove the handling in `player_event_handler.rs` again and only let it be an library feature?
3. Or we just go the easiest way and add an parameter that configure and enable the feature.

I would be fine with any of the previously mentioned solutions. From a maintenance perspective I would prefer the second solution. @kingosticks any preference from your side?

Reply:
I'm fine either. I would wait for your internal alignment to not commit and revert something :)

Reply:
I suggest making it a library feature for now, because:

1. We'll find out when users come knocking when they want it in the binary anyway
2. If we're to take moOde as one example, I know it doesn't show progress for any of its external renderers.

Reply:
Done. I did remove handling and pass None as option in the binary. :)